contents/handbook/company/team/joe-martin-goals.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Joe Martin's goals
+sidebar: Handbook
+showTitle: true
+---
+
+This is a non-exhaustive list of things I'm currently trying to accomplish. It's likely that ad-hoc tasks or other priorities will appear to alter these and that's OK. For information on how I like to work, refer to [my README](https://posthog.com/handbook/company/team/joe-martin). 
+
+
+### Execution - Q4 2021
+- **Content:** Get the content pipeline working properly, with a clear content calendar. Execute against the content strategy, set the Technical Content Marketer up for success. 
+- **Landing Pages:** Work with Team Design to [get a usable landing page template made](https://github.com/PostHog/posthog.com/issues/2098) and create competitor landing pages, with further pages planned against the content strategy. Identify SEO opportunities with landing pages and hit some of the big ones.
+- **Case studies:** Get [the customer stories section](https://github.com/PostHog/posthog.com/issues/1707) of the site live and continue to build on them with [Crypto and Healthcare content](https://github.com/PostHog/posthog.com/issues/2092). Bring case study content more into the forefront across PostHog documentation and webpages. 
+
+### Strategic - Q4 2021
+- **Content:** Ensure Team Marketing has a content strategy in place and a plan on how to deliver against it. 
+- **Competitors & Churn:** Invest time in competitor analysis and churned customer feedback, using discoveries to influence other teams and improve messaging. Focus on up-and-coming competitors where possible. 
```

Comment:
Would also love your support on where product should focus in 2022 - especially in terms of competitor analysis and any external trends / learnings.